modules/price/src/Resolver/DefaultPriceResolver.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+namespace Drupal\commerce_price\Resolver;
+
+use Drupal\commerce\PurchasableEntityInterface;
+use Drupal\commerce_price\Plugin\Field\FieldType\Price;
+
+/**
+ * Returns the price based on the purchasable entity's price field.
+ */
+class DefaultPriceResolver implements PriceResolverInterface {
+
+  /**
+   * {@inheritdoc}
+   */
+  public function resolve(PurchasableEntityInterface $entity, $quantity = 1) {
+    return $this->applies($entity) ? $entity->price->first()->toPrice() : NULL;
```

Comment:
We have no guarantee $entity->price exists. Let's add getPrice() to PurchasableEntityInterface. We can copy it from the variation interface.